assets/airports/kcvg.json

```diff
@@ -0,0 +1,6856 @@
+        },
+        {
+            "origin": "KCVG",
+            "destination": "",
+            "category": "departure",
+            "route": "KCVG.KENLN4.HYK",
+            "altitude": null,
+            "speed": null,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "KCVG",
+            "destination": "",
+            "category": "departure",
+            "route": "KCVG.LOVEY6.AXC",
+            "altitude": null,
+            "speed": null,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "KCVG",
+            "destination": "",
+            "category": "departure",
+            "route": "KCVG.LOVEY6.CADIZ",
+            "altitude": null,
+            "speed": null,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "KCVG",
+            "destination": "",
+            "category": "departure",
+            "route": "KCVG.ROCKT8.AHTIY",
+            "altitude": null,
+            "speed": null,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "KCVG",
+            "destination": "",
+            "category": "departure",
+            "route": "KCVG.ROCKT8.TVT",
+            "altitude": null,
+            "speed": null,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "KCVG",
+            "destination": "",
+            "category": "departure",
+            "route": "KCVG.SILKS4.TRFWA",
+            "altitude": null,
+            "speed": null,
+            "method": "random",
+            "rate": 1,
+            "airlines": [
+                ["dal", 51],
+                ["aal", 17],
+                ["ual", 11],
+                ["aca", 2],
+                ["swa", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KCVG",
+            "category": "arrival",
+            "route": "DELHI.CEGRM4.KCVG",
```

Comment:
Remove one of the spawnPatterns for the CERGM4 arrival as they will meet at some point and cause seperation issues.